app.rb

```diff
@@ -0,0 +1,18 @@
+require 'sinatra/base'
+
+class Chitter < Sinatra::Base
+
+	get '/' do
+		erb(:index)
+	end
+	
+	get '/login' do
+		erb(:login)
+	end
+
+	get '/signup' do
+		erb(:signup)
+	end
```

Comment:
Controllers are skinny and it's clear what each route does from it's name - tab length seems large (6?)